doc/tech/approach.md

```diff
@@ -80,27 +80,19 @@ To address the concern areas detailed above the specific topics listed below wil
 
 This section introduces some networking related principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architecture 01 (OpenStack), Reference Architecture 02 (K8s) and down to Reference Implementations and Reference Conformance test suites under CNTT.
```

Comment:
for consistency in use of uppercase and lowercase as well as some suggestion on language (without a "from" use of "down to"):
```suggestion
This section introduces some networking related principles that should be followed during development and definition of the networking scope for Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites.
```